Adding alt text to several images in several files. (#79)
mulesoft/docs-apikit: modules/ROOT/pages/creating-an-apikit-4-project-with-maven.adoc

. In Studio, under the *File* menu, select *Import*.
. In the Import wizard, expand the *Mule* folder, then select `Maven-based Mule project from pom.xml`.
+

> image::import-maven.png["Import window highlights the Mule folder, in which the pom.xml file should be selected."]

Reviewer: ```suggestion
image::import-maven.png["Import window highlights the Mule folder, in which to select the pom.xml file."]
```
